compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -93,6 +99,11 @@ class SkikoComposeUiTest(
     private val infiniteAnimationPolicy = object : InfiniteAnimationPolicy {
         override suspend fun <R> onInfiniteOperation(block: suspend () -> R): R {
             if (mainClock.autoAdvance) {
+                if (throwOnInfiniteAnimation) {
+                    error("Infinite animations are disabled on tests with mainClock.autoAdvance=true. " +
```

Comment:
It's helpful to find tests where infinite animations are ignored, so they don't really make any UI change (the capture bitmap is not what we expect)

Reply:
Is it only temporary solution? I think, we shouldn't deviate from Android implementation, changing the default behavior, and allowing configuring this more. Maybe `autoAdvance` is enough? We can describe, that with `autoAdvance`, infinite animations only have the first frame.

Maybe it is more correct two have 2 properties for 2 behaviors (auto advancing and stopping infinite animations), but I would change the name to something like `stopInfiniteAnimations`, and commonize this behavior. But do it as a separate feature in the future for all targets at once.

Reply:
right, I didn't think about deviating from android behaviour. 
I'll remove it here for now.